config.yml

```diff
@@ -1,27 +1,53 @@
 state_abbr: 'IL'
 
 geo_res: 'rto'  # accepts: 'rto' or 'county'
-time_res: 4  # hours 
+time_res: 1  # hours 
+load_filter: 60e3  # MW, load above this level will be removed as outliers.
+total_demand: 136e6  # Annual MWh in the first year
+load_growth: 0.00  # % annual growth
+load_share:
```

Comment:
How/where is this used?

Reply:
this parameter is not used, anywhere. I think I intended to assign a fraction of the total demand to each bus region, but it was unnecessary.